### ISSUE TITLE
Cannot start backend services with `make start` unless _all_ services are started

### ISSUE THREAD

**Issue by scubbo** [grafana/oncall#846]
System information:
* Macbook Pro, Mac OS 12.6, Intel Core i7
* Docker compose version 2.7.0

Steps to reproduce:
```
$ git clone git@github.com:grafana/oncall.git
Cloning into 'oncall'...
[...]

$ cd oncall

$ git show --pretty=oneline -s
41f8d4afb19ef1217663bbd94180602d9b3767eb (HEAD -> dev, origin/dev, origin/HEAD) PD migrator: add support for email notifications (#839)

$ make init
[...]
✔ Compiling...
✨  Done in 45.96s.

$ make start
COMPOSE_PROFILES=engine,oncall_ui,redis,grafana DB=sqlite BROKER_TYPE=redis SQLITE_DB_FILE=./engine/oncall.db docker compose -f docker-compose-developer.yml up --remove-orphans -d
[+] Running 5/6
 ⠿ Container grafana              Created                                                  0.1s
 ⠿ Container oncall_ui            Started                                                 22.9s
 ⠿ Container redis                Waiting                                                 23.0s
 ⠿ Container oncall_db_migration  Created                                                  0.1s
 ⠿ Container oncall_celery        Created                                                  0.1s
 ⠿ Container oncall_engine        Created                                                  0.1s
no such service: postgres
make: *** [start] Error 1
# Note - I can repeat this command and it will error on either `postgres` or `mysql`, apparently randomly

$ make start COMPOSE_PROFILES=engine,oncall_ui,redis,grafana,postgres
[...]
no such service: mysql
make: *** [start] Error 1
make start COMPOSE_PROFILES=engine,oncall_ui,redis,grafana,postgres,mysql
COMPOSE_PROFILES=engine,oncall_ui,redis,grafana,postgres,mysql DB=mysql BROKER_TYPE=redis docker compose -f docker-compose-developer.yml up --remove-orphans -d
[+] Running 10/10
 ⠿ Container oncall_ui                      Running                                        0.0s
 ⠿ Container mysql                          Healthy                                        1.0s
 ⠿ Container redis                          Running                                        0.0s
 ⠿ Container mysql_to_create_grafana_db     Star...                                        1.8s
 ⠿ Container postgres                       Healthy                                        1.0s
 ⠿ Container postgres_to_create_grafana_db  S...                                           1.9s
 ⠿ Container oncall_db_migration            Recreated                                      0.2s
 ⠿ Container grafana                        Started                                        1.9s
 ⠿ Container oncall_engine                  Recreated                                      0.2s
 ⠿ Container oncall_celery                  Recreated                                      0.2s
no such service: rabbitmq
make: *** [start] Error 1

$ make start COMPOSE_PROFILES=engine,oncall_ui,redis,grafana,postgres,mysql,rabbitmq
[services start up successfully, grafana is available on localhost:3000/login]
```

**Comment by joeyorlando** [comment]
@scubbo thanks for reporting.

If you upgrade your `docker-compose` to the latest version (`v2.12.1`), are you still able to reproduce this? I _believe_ that this issue was patched in `v2.10` of `docker-compose` ([CHANGELOG](https://docs.docker.com/compose/release-notes/#bug-fixes-6); related `docker-compose` [issue](https://github.com/docker/compose/issues/9591)).

**Comment by scubbo** [comment]
```
$ git clone git@github.com:grafana/oncall.git oncall-fresh-clone
Cloning into 'oncall-fresh-clone'...
[...]
$ cd oncall-fresh-clone
$ docker-compose version
Docker Compose version v2.12.2
$ make init start
[...]
COMPOSE_PROFILES=engine,oncall_ui,redis,grafana DB=sqlite BROKER_TYPE=redis SQLITE_DB_FILE=./engine/oncall.db docker compose -f docker-compose-developer.yml up --remove-orphans -d
[...]
[+] Running 9/9
 ⠿ Network oncall_dev                           Created                                                           0.1s
 ⠿ Volume "oncall-fresh-clone_grafanadata_dev"  Created                                                           0.0s
 ⠿ Volume "oncall-fresh-clone_redisdata_dev"    Created                                                           0.0s
 ⠿ Container oncall_ui                          Started                                                          13.9s
 ⠿ Container redis                              Healthy                                                          19.5s
 ⠿ Container grafana                            Started                                                          13.8s
 ⠿ Container oncall_db_migration                Exited                                                           51.8s
 ⠿ Container oncall_engine                      Started                                                          52.5s
 ⠿ Container oncall_celery                      Started                                                          52.5s
```

Closing. Thanks for the tip! I'll make a change to `./dev/README.md`
